clients/vscode-hlasmplugin/src/hlasmDownloadCommands.ts

```diff
@@ -251,46 +251,133 @@ function fixPath(p: string): string {
     return p;
 }
 
-function getWasmRuntimeArgs(): Array<string> {
-    const v8Version = process?.versions?.v8 ?? "1.0";
-    const v8Major = +v8Version.split(".")[0];
-    if (v8Major >= 10)
-        return [];
-    else
-        return [
-            '--experimental-wasm-eh'
-        ];
-}
-
-async function unterse(outDir: string): Promise<{ process: Promise<void>, input: Writable }> {
+export async function unterse(outDir: string): Promise<{ process: Promise<void>, input: Writable }> {
```

Comment:
I understand that `input` hear means a stream that serves as an input into the processing. But having a writable stream called `input` is a bit confusing. What about alling it `stream`? From the type it is clear that it is a `Writable`.

That would make it a `Writable` `stream` which I think is clearer from the consumer perspective.